nva-datamodel-java/src/main/resources/publication-ontology.ttl

```diff
@@ -844,7 +844,7 @@ nva:JournalReview a rdfs:Class ;
   rdfs:subClassOf nva:Journal .
 
 nva:PublicationDate a rdfs:Class ;
-  rdfs:comment "" ;
+  rdfs:comment "When the publication was first published in any system or form" ;
```

Comment:
I think it is "The date the thing was first published for public consumption", using the word "publication" is problematic since it relates closely to "Publication", which PublicationDates are never directly associated with.

Reply:
Your suggestion is so much better than mine, thank you.